### DISCUSSION TITLE
How do I make Renovate bot to update only LTS versions of nodejs

### DISCUSSION THREAD

**Discussion by tal-rofe** [renovatebot/renovate#14643] [Request Help]
I have configured Renovate with` renovate.config.js`:
```
{
	"extends": ["config:base", ":semanticCommitTypeAll(chore)"],
	"packageRules": [
		{
			"updateTypes": ["minor", "patch"],
			"automerge": true
		}
	]
}
```

Moreover, I've added the following file (dedicated to this question purpose) called `.nvmrc`:
`lts/*
`

I expected Renovate to upgrade my repository used Node.js versions to only LTS versions (so basically 16) but it updated to 17.
For example - it tried the following:
`16-alpine -> 17-alpine`

I guess I have some incorrect configuration?

How can I make Renovate to update only LTS versions of Node.js?

**Comment by rarkins** [comment]
Not possible due to the -alpine suffix. Our node versioning understands LTS but not suffixes like that.

**Reply by tal-rofe** [reply]
@rarkins 
Is it something that will be supported in the future?

**Reply by viceice** [reply]
I think there is an open issue about it.

**Reply by rarkins** [reply]
#5990

**Reply by ponchofiesta** [reply]
@rarkins Renovate now understands the suffix (I assume). Is it now only possible to update to LTS versions?

**Reply by rarkins** [reply]
You can now use `versionCompatibility` to configure this. Make `-alpine` the compatibility part
